### ISSUE TITLE
Cannot call sqlite3.backup(db) on a TracedSQLite object

### ISSUE THREAD

**Issue by JLHasson** [DataDog/dd-trace-py#4241]
Thanks for taking the time for reporting an issue!

Before reporting an issue on dd-trace-py, please be sure to provide all
necessary information.

If you're hitting a bug, make sure that you're using the latest version of this
library.

### Which version of dd-trace-py are you using?
1.5.0
### Which version of pip are you using?
21.1.1
_ddtrace requires pip>=18 to install one of our pre-built wheels_

### Which version of the libraries are you using?

You can copy/paste the output of `pip freeze` here.

```
ddtrace==1.5.0
```

### How can we reproduce your problem?

```
from ddtrace import config, patch_all
import sqlite3

config.env = "test"  # the environment the application is in
config.service = "app"  # name of your application
config.version = "v1"  # version of your application
patch_all()

src = sqlite3.connect("1.db")
dst = sqlite3.connect("2.db")
with dst:
    src.backup(dst, pages=1)
dst.close()
src.close()
```

### What is the result that you get?

The following TypeError
```
TypeError: backup() argument 1 must be sqlite3.Connection, not TracedSQLite
```

### What is the result that you expected?

The function should succeed without error.

**Comment by brettlangdon** [comment]
@JLHasson thank you for reporting, I have opened up #4246 to hopefully address this issue.

**Comment by JLHasson** [comment]
Thanks for the quick fix! How does the release process for dd-trace work, when will we see this in pypi?

**Comment by JLHasson** [comment]
Also, I'm not exactly sure the correct place to report this, but this honestly feels like a but in `sqlite3`. IMO instead of explicitly checking for a `sqlite3.Connection`, they should be doing the equivalent of `isinstance(sqlite3.Connection, o)` which should then work with a `TracedSQLite` since I assume it subclasses `sqlite3.Connection`?

**Comment by brettlangdon** [comment]
> Thanks for the quick fix! How does the release process for dd-trace work, when will we see this in pypi?

Once the fix PR is merged we will backport and do a bug fix release (currently would be 1.5.1).


> Also, I'm not exactly sure the correct place to report this, but this honestly feels like a but in sqlite3

Personally I am not sure either, but it is fairly common to do strict type checking in C extensions.

https://discuss.python.org/ might be a great place to raise it if you wanted to!

**Comment by brettlangdon** [comment]
@JLHasson this is now released in 1.5.1.

Sorry for the delay we had some CI issues which delayed this release.

**Comment by JLHasson** [comment]
Great! Thanks for the update, will try the updated version next week 💯
